lib/nimble_options.ex

```diff
@@ -440,6 +442,31 @@ defmodule NimbleOptions do
     end
   end
 
+  defp validate_type({:list, type}, key, values) when is_list(values) do
+    Enum.reduce_while(values, :ok, fn value, acc ->
+      case validate_type(type, key, value) do
+        :ok ->
+          {:cont, acc}
+
+        {:ok, _} ->
+          {:cont, acc}
+
+        _ ->
```

Comment:
I'm a bit worried about throwing this away. Again, if the validator is `{:custom, mod, fun, args}`, the message could contain detail on how to fix the error, but we're discarding it here. 🤔

Reply:
Yeah that's a mistake on the code :)

https://github.com/dashbitco/nimble_options/pull/35#discussion_r423810384